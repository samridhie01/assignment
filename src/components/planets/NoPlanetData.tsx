import React from "react";
import {getSearchKey} from "../utils/utils";

const NoPlanetData = ()=>{
    return(
        <h3 data-testid={"search-update"}>no planet with climate {getSearchKey()}</h3>
    )
}

export default  NoPlanetData;
