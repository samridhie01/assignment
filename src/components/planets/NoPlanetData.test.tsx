import {render} from "@testing-library/react";
import React from "react";
import NoPlanetData from "./NoPlanetData";

describe("Search Box", ()=>{
    it('should render snapshot', () => {
        const { asFragment } = render(
            <NoPlanetData />
        );
        expect(asFragment()).toMatchSnapshot();
    });
})
