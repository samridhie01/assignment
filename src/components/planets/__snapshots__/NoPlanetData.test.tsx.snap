// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Box should render snapshot 1`] = `
<DocumentFragment>
  <h3
    data-testid="search-update"
  >
    no planet with climate 
  </h3>
</DocumentFragment>
`;
